File: Beginner/go/1065.go
// 
package main

import "fmt"

func main() {
	var t, count int
	t = 5
	count = 0
	for i := 1; i <= t; i++ {
		var n int
		fmt.Scan(&n)
		if n % 2 == 0 {
			count++
		}
	}
	fmt.Printf("%d valores pares\n", count)
}

File: Beginner/cpp/1065.cpp
// Write a program 

#include <bits/stdc++.h>
using namespace std;

int main() {
    int t = 5, count = 0;

    while(t--){
        int n;
        cin >> n;

        if(n % 2 == 0) count++;
    }

    cout << count << " valores pares" << endl;
return 0;
}